Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent any
+
+    environment{
+        JOB= '$JOB'
+        SIM= '$SIM'
+        PIPELINE= '$PIPELINE'
+        EXPECTED_FAILURES= '$EXPECTED_FAILURES'
+        EXTRA_CORE_ARGS= '$EXTRA_CORE_ARGS'
+    }
+    stages {
+        stage('Build') {
+            steps {
+                sh 'docker pull librecores/librecores-ci:0.2.0'
+                sh 'docker images'
+            }
+        }
+        stage('Run') {
+            steps {
+                sh 'docker run --rm -v $(pwd):/src -e JOB -e SIM -e PIPELINE -e EXPECTED_FAILURES -e EXTRA_CORE_ARGS librecores/librecores-ci /src/.travis/test.sh'
+            }
+        }
+        stage('run-parallel-jobs') {
+            steps {
+                parallel(
```

Comment:
We could have definitely optimize it if we had native support of config matrixes in Declarative pipeline, but it s still in progress

Reply:
> Has it been tested locally @Nancy-Chauhan ? I can enable CI for your local branch on ci.librecores.org f you grant @librecores-ci-bot Write access to your repo

Yes , I have tested locally , still need to do changes and I have send invite to librecores-ci-bot for  write access